spark/src/main/scala/geotrellis/spark/filter/Implicits.scala

```diff
@@ -0,0 +1,10 @@
+package geotrellis.spark.filter
+
+import geotrellis.spark.SpaceTimeKey
+
+
+object Implicits extends Implicits
+
+trait Implicits {
+  implicit def SpaceTimeToSpatial(st: SpaceTimeKey) = st.spatialKey
```

Comment:
No need to have this (can use `SpatialComponent`).

This is where the implicit classes would live, then root package extends this trait